### PR TITLE
Save should use submit

### PR DESCRIPTION
**Pull request by jtnord**
builds upon #534 and #535 but does not need any of them (I was just too lazy to rebase as I have other WIP in the same tree)

6faa6c878248846c1227955175166a2b098a5262 is the isolated changes for this PR.

### CODE REVIEW

**Comment by jtnord** [comment]
Want to try this locally as well as the CI (so marking as WIP) because last time I tried to use `submit()` it all went Pete Tong.

**Comment by jtnord** [comment]
this should fix an observed issue where the configuration had not been saved whilst an attempt to do another operation was performed (that relied on the configuration being saved first)

**Comment by amuniz** [comment]
Can you rebase to get this change alone? (upstream PRs are merged)

**Comment by jtnord** [comment]
no need to revise, just GitHub is not smart enough..n

**Comment by jtnord** [comment]
@amuniz tada!

File: src/main/java/org/jenkinsci/test/acceptance/po/ConfigurablePageObject.java [hunk 1]
```diff
@@ -126,8 +130,10 @@ public void ensureConfigPage() {
     public abstract URL getConfigUrl();
 
     public void save() {
-        clickButton("Save");
+        WebElement e = find(by.button("Save"));
+        e.submit();
```
Comment (amuniz) [inline src/main/java/org/jenkinsci/test/acceptance/po/ConfigurablePageObject.java]: Seems weird to call `submit` on a generic `WebElement`, but if it works...

**Pull request merged**